compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ParameterDesugar.java

```diff
@@ -1464,7 +1463,8 @@ public void visit(BLangXMLSequenceLiteral xmlSequenceLiteral) {
     @Override
     public void visit(BLangRegExpTemplateLiteral regExpTemplateLiteral) {
         List<BLangExpression> interpolationsList =
-                symResolver.getListOfInterpolations(regExpTemplateLiteral.reDisjunction.sequenceList);
+                symResolver.getListOfInterpolations(regExpTemplateLiteral.reDisjunction.sequenceList,
```

Comment:
Is this change related to IDL Client Change?

Reply:
This is related to reg exp.

Reply:
Regex related changes are sorted.
please chec @gimantha @lasinicl